conftest.py

```diff
@@ -45,6 +45,22 @@ def mock_env_passport_url():
     env["PASSPORT_URL"] = original_passport_url
 
 
+@pytest.fixture(scope="function")
+def mock_env_secret_key():
+    # Set the environment variable SECRET_KEY to the mocked value
+    original_secret_key = env.get("SECRET_KEY", "")
+
+    with open("security/test_keys/001.key", "r") as key_file:
+        private_key = key_file.read()
+    env["SECRET_KEY"] = private_key
```

Comment:
Sure we can set two keys `SECRET_KEY` and `PKI_PRIVATE_KEY`.
I used only SECRET_KEY because in either JOSE or IETF signing we use a private key, not a string. So if we have two  env variables I guess we will not use SECRET_KEY at all? 🤔